Prefer fetched configurations using SSL over plaintext.

--- a/modules/accountcreation/readFromXML.sys.mjs       2010-01-01 00:00:00.000000000 +0100
+++ b/accountcreation/readFromXML.sys.mjs    2024-11-06 10:08:18.052090360 +0100
@@ -102,7 +102,10 @@
             SSL: Ci.nsMsgSocketType.SSL,
             STARTTLS: Ci.nsMsgSocketType.alwaysSTARTTLS,
           });
-          break; // take first that we support
+          if (iO.socketType != Ci.nsMsgSocketType.plain) {
+            // pick first non-plaintext protocol, if available
+            break;
+          }
         } catch (e) {
           exception = e;
         }
@@ -225,7 +228,10 @@
             SSL: Ci.nsMsgSocketType.SSL,
             STARTTLS: Ci.nsMsgSocketType.alwaysSTARTTLS,
           });
-          break; // take first that we support
+          if (oO.socketType != Ci.nsMsgSocketType.plain) {
+            // pick first non-plaintext protocol, if available
+            break;
+          }
         } catch (e) {
           exception = e;
         }
